debian/preinst

```diff
@@ -9,6 +9,13 @@ set -e
 
 CONFIG="/etc/apt/apt.conf.d/50unattended-upgrades"
 
+# Retrieves (mostly user-defined) repositories in /apt/lists 
+# and configures for automatic updates.
+missingSources=$(python listMissingRepos.py)
+for source in $missingSources; do
+    sed -i "/Unattended-Upgrade::Allowed-Origins {/a \\\t$source" $CONFIG
```

Comment:
Handling configuration files is not that easy. If you would like to contribute to Debian packages in that area please read about dpkg's configuration file handling mechanism and ucf.